includes/admin/class-wcmp-admin.php

```diff
@@ -491,41 +533,42 @@ public function barcode_add_new_order_admin_list_column($columns)
 
     /**
      * @param $column
+     *
+     * @throws Exception
      */
-    public function barcode_add_new_order_admin_list_column_content($column)
+    public function addBarcodeToOrderColumn($column)
     {
         global $post;
 
-        if ('barcode' === $column) {
-            $order = WCX::get_order($post->ID);
-            echo $this->get_barcode($order);
+        if ("barcode" === $column) {
+            $this->get_barcode(WCX::get_order($post->ID));
         }
     }
 
     /**
-     * @param      $order
-     * @param null $barcode
+     * @param WC_Order $order
      *
-     * @return string|null
+     * @return void
      * @throws Exception
      */
-    public function get_barcode($order, $barcode = null)
+    public function get_barcode(WC_Order $order): void
     {
         $shipments = WCMP_Admin::get_order_shipments($order, true);
 
         if (empty($shipments)) {
-            return __("No label has been created yet.", "woocommerce-myparcelbe");
+            echo __("No label has been created yet.", "woocommerce-myparcelbe");
+            return;
```

Comment:
Kan je dit echt niet returnen? Het is niet echt netjes om een echo plaatsen in get_barcode(). Maar als het teveel werk is, laten we het zo.

Reply:
Het is ook helemaal geen get functie, ik heb hem renderBarcode genoemd.